src/FormData.ml

```diff
@@ -0,0 +1,53 @@
+(* Provides a way to easily construct a set of key/value pairs representing form fields and their values, which can then be easily sent using the XMLHttpRequest.send() method. It uses the same format a form would use if the encoding type were set to "multipart/form-data". *)
+
+module Iterator = struct
+  type 'a iterator
+  type 'a t = 'a iterator
+  type 'a value = { done_ : bool option; [@mel.as "done"] value : 'a option }
+
+  external next : 'a t -> 'a value = "next" [@@mel.send]
+  external toArray : 'a t -> 'a array = "Array.from"
+  external toArrayWithMapper : 'a t -> f:('a -> 'b) -> 'b array = "Array.from"
+end
+
+type t
+type file
+type blob
+type entryValue
+
+let classify : entryValue -> [> `String of string | `File of file ] =
```

Comment:
I think we could just use `Js.Types.classify` as I mentioned in the melange PR comment.